moto/iot/urls.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import unicode_literals
+from .responses import IoTResponse
+
+url_bases = [
+    "https?://iot.(.+).amazonaws.com",
+]
+
+
+response = IoTResponse()
+
+
+url_paths = {
+    '{0}/.*$': response.dispatch,
```

Comment:
is response used in that module elsewher or would `IoTResponse().dispatch` do?

Reply:
Just writing like will works.
```
url_paths = {
     '{0}/.*$':IotResponse().dispatch
}
```

But scaffold script generates `urls.py` with `response = {{ service_class }}Response()` at the first generation of each aws service when the API protocol is `rest-json`.
So, it will be easy to keep this as is.

Reply:
Just writing like below will works.
```
url_paths = {
     '{0}/.*$':IotResponse().dispatch
}
```

But scaffold script generates `urls.py` with `response = {{ service_class }}Response()` at the first generation of each aws service when the API protocol is `rest-json`.
So, it will be easy to keep this as is.

And I have no time to fix scaffold script correctly now 😢